doc/api/n-api.md

```diff
@@ -34,13 +34,53 @@ properties:
   handling section [Error Handling][].
 
 The N-API is a C API that ensures ABI stability across Node.js versions
-and different compiler levels. However, we also understand that a C++
-API can be easier to use in many cases. To support these cases we expect
-there to be one or more C++ wrapper modules that provide an inlineable C++
-API. Binaries built with these wrapper modules will depend on the symbols
-for the N-API C based functions exported by Node.js. These wrappers are not
-part of N-API, nor will they be maintained as part of Node.js. One such
-example is: [node-addon-api](https://github.com/nodejs/node-addon-api).
+and different compiler levels. **However, a C++ API can be easier to use
+in many cases.** In order to support using C++ the project maintains a
+C++ wrapper module called
+[node-addon-api](https://github.com/nodejs/node-addon-api).
+This wrapper provides an inlineable C++ API. Binaries built
+with `node-addon-api` will depend on the symbols for the N-API C based
+functions exported by Node.js. `node-addon-api` can be seen as a more
+efficient way to write code that calls N-API. For example if the following
+`node-addon-api` code is written:
+
+```C++
+Object obj = Object::New(env);
+obj["foo"] = String::New(env, "bar");
+```
+
+what actually gets used in the addon is:
```

Comment:
Splitting the sentence up with examples and having this fragment here makes it pretty hard to read/understand IMO.